include/semaphore.h

```diff
@@ -48,8 +48,8 @@
 
 /* Bit definitions for the struct sem_s flags field */
 
-#define PRIOINHERIT_FLAGS_DISABLE (1 << 0)  /* Bit 0: Priority inheritance
-                                             * is disabled for this semaphore. */
+#define PRIOINHERIT_FLAGS_ENABLE (1 << 0)  /* Bit 0: Priority inheritance
```

Comment:
I do not see where `NXMUTEX_INITIALIZER` and `NXRMUTEX_INITIALIZER` are updated to reflect this change.

Reply:
let change NXMUTEX_INITIALIZER(xxx, PRIOINHERIT_FLAGS_DISABLE) to MUTEX_INITIALIZER(xxx)